src/components/shared-tabs.scss

```
@@ -128,8 +128,12 @@
     overflow-x: auto;
     scroll-snap-type: x mandatory;
     scroll-padding-inline: 4px;
-    padding: 2px 4px 6px;
-    margin-inline: -4px;
+    // overflow-x:auto implicitly clips overflow-y. The active card's
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)